projects/swimlane/ngx-ui/src/lib/components/plus-menu/plus-menu.component.scss

```diff
@@ -0,0 +1,551 @@
+@import 'colors/variables';
+
+.ngx-plus-menu {
+  --menu-color: #9fce36;
+  --background-color: 0, 0, 0;
+  --background-opacity: 0.7;
+
+  --size: 300px;
+  --radial-size: 500px;
+  --radial-radius: 250px;
+
+  --button-size: 30px;
+  --button-radius: 15px;
+
+  --icon-size: 60px;
+  --icon-background: linear-gradient(180deg, #12141a 0%, #262c38 100%);
+
+  --shadow-blur: 50px;
+
+  --animation-duration: 500ms;
+
+  width: var(--button-size);
+  height: var(--button-size);
+
+  transition-property: height width background box-shadow border-radius translate;
+  transition-duration: var(--animation-duration);
+
+  background: rgba(var(--background-color), 0);
+  box-shadow: 0 0 var(--shadow-blur) var(--shadow-blur) rgba(var(--background-color), 0);
+  border-radius: var(--button-radius);
+  overflow: hidden;
+  z-index: 10;
+
+  &--circle i {
+    transform: rotate(0deg);
+    transition: transform var(--animation-duration);
+  }
+
+  &.open {
+    z-index: 100;
+
+    .ngx-plus-menu--circle i {
+      transform: rotate(135deg);
+    }
+  }
+
+  &--circle-container {
+    color: var(--menu-color);
+    cursor: pointer;
+    font-size: 29px;
+    user-select: none;
+  }
+
+  &--menu-title {
+    display: none;
+    vertical-align: top;
+    font-size: 18px;
+    color: white;
+    margin-left: 5px;
+    line-height: var(--button-size);
+  }
+
+  &--content-container > svg {
+    position: absolute;
+    pointer-events: none;
+
+    > g {
+      display: none;
+    }
+
+    .arrow {
+      fill: var(--menu-color);
+    }
+
+    .arc {
+      stroke: var(--menu-color);
+    }
+  }
+
+  &--items-container {
+    position: absolute;
+    top: var(--button-size);
+    right: var(--button-size);
+
+    height: var(--radial-size);
+    width: var(--radial-size);
+    color: white;
+
+    .ngx-plus-menu--item {
+      position: absolute;
+      width: 90px;
+      height: auto;
+      text-align: right;
+      font-size: 18px;
+      line-height: 19px;
+      font-weight: bold;
+      cursor: pointer;
+
+      .keys {
+        font-style: normal;
+        font-weight: 600;
+        font-size: 12px;
```

Comment:
Let's bump the font size to 13px. There's a rendering difference between Figma and web fonts, and small fonts sometimes need to be bumped up—even though you implemented exactly as represented in design.